web-ide/web_ide.py

```diff
@@ -168,9 +169,19 @@ async def on_shutdown():
     if dirpath.exists():
         shutil.rmtree(dirpath)
 
+async def export(q: Q):
+    tmp_files = [os.path.join(tmp_dir, file) if file.endswith(('.py')) else None for file in os.listdir(tmp_dir)]
+    with ZipFile('tmp_project/app.zip','w') as zip:
+        for file in tmp_files:
+            zip.write(file)
+    zip_path = await q.site.upload(['tmp_project/app.zip'])
+    webbrowser.open(f"{_server_adress}{zip_path[0]}", autoraise=True)
```

Comment:
We need user's browser to download the exported file, not the server browser. The current solution will work only locally, but not in production.

Just set the uploaded zip path to ui.button/link for user to download.